x-pack/plugins/fleet/server/services/epm/package_service.ts

```diff
@@ -139,7 +151,9 @@ class PackageClientImpl implements PackageClient {
     pkgVersion?: string;
     spaceId?: string;
   }): Promise<Installation | undefined> {
-    await this.#runPreflight();
+    await this.#runPreflight({
+      integrations: { installPackages: true },
```

Comment:
TODO: should be moved to a constant to use the [same as the API](https://github.com/elastic/kibana/blob/169d197d1a219ced2ca95a0de639f3b6d465b2a5/x-pack/plugins/fleet/server/routes/epm/index.ts#L165)